dotcom-rendering/src/web/components/ArticleHeadline.ServerTestsDemo.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { ArticleDesign, ArticleDisplay, ArticlePillar } from '@guardian/libs';
+import { ArticleContainer } from './ArticleContainer';
+import { ArticleHeadline } from './ArticleHeadline';
+import { Flex } from './Flex';
+import { LeftColumn } from './LeftColumn';
+import { Section } from './Section';
+import {
+	mockServerSideTestsProviderFactory,
+	useServerSideTests,
+} from './ServerSideTestProvider';
+
+export default {
+	component: ArticleHeadline,
+	title: 'Components/ArticleHeadlineServerTestDemo',
+};
+
+const format = {
+	display: ArticleDisplay.Standard,
+	design: ArticleDesign.Standard,
+	theme: ArticlePillar.News,
+};
+
+const ArticleHeadlineWithTest = () => {
```

Comment:
I'm creating this component in the story file just to demonstrate what a component with a test would look like. We can imagine that this is a regular component in the `components/` directory which has had test code added to it.